### ISSUE TITLE
[Snackbar] Delay in render

### ISSUE THREAD

**Issue by leebyp** [uber/baseweb#4771]
Hi all, I've been using the Snackbar API for user feedback per the recommendations in the [docs](https://baseweb.design/components/snackbar/#snackbar), but I noticed that there is a ~2s delay in the render - can be reproduced directly on the docs page too. Is there functionality to customize the time here / is this a bug?

**Comment by Jassi10000** [comment]
First I want you to check your network connnection too, as may be due to slow network, rendering might had been slow
Else it can be a bug / might be the rendering is set to be dene at ~2s

**Comment by williamernest** [comment]
Hmm, interesting. It looks like it's waiting a second before starting the transition, and then the transition takes an additional second. Probably not intentional, as I would expect it to be more seamless (animation starts shortly after clicking the button). Thanks for filing the issue.

**Comment by brycedorn** [comment]
Good catch! I've noticed this too & looked into it - there is indeed a [1s timeout](https://github.com/uber/baseweb/blob/master/src/snackbar/snackbar-context.js#L96) when displaying a toast which I think is a bug.

Made a pr^ for this, let me know if the delay value should be a prop instead? I'd assume the expected behavior would be to show it immediately though.
